src/api/models/publishedDataset.js

```diff
@@ -106,7 +108,14 @@ export default (db) => {
         )({ removedAt: { $exists: false } });
         const cursor = collection.getFindCursor(filters, sortBy, sortDir);
 
-        return cursor.stream();
+        return cursor
+            .map(resource => (
```

Comment:
No because nothing prevent Julia to change the instance config after publication